transport/transport.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import tarfile
+
+from distutils.util import strtobool
+
+import ci.util
+import processing.config as config
+import processing.processing as processing
+import processing.processing_component as pc
+
+
+def parse_args():
+    parent_parser = argparse.ArgumentParser(
+        description='Transport tool for component descriptor v2',
+    )
+
+    subparsers = parent_parser.add_subparsers(dest="action", help="Actions", required=True)
+    parser_download = subparsers.add_parser(
+            name="download",
```

Comment:
might make sense to extract the actions to an enum (since they are being checked for later)

Reply:
definitely makes sense ;)